gatsby-config.js

```diff
@@ -117,6 +117,16 @@ module.exports = {
             },
           },
         },
+        tessen: {
+          product: 'DOC',
+          subproduct: 'TDOC',
+          segmentWriteKey: 'AEfP8c1VSuFxhMdk3jYFQrYQV9sHbUXx',
```

Comment:
I assume this is safe to commit?

Reply:
Yep! It's used as a way to let segment know where to put data ([docs](https://segment.com/docs/connections/find-writekey/)). This was taken straight from the current docs site.

Reply:
yep. i checked way back when if it was okay that this was exposed, and security signed off on it